Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealitySpeechCommandsProfileInspector.cs

```diff
@@ -50,6 +53,18 @@ public override void OnInspectorGUI()
 
             EditorGUILayout.HelpBox("Speech Commands are any/all spoken keywords your users will be able say to raise an Input Action in your application.", MessageType.Info);
 
+            if (!MixedRealityManager.Instance.ActiveProfile.IsInputSystemEnabled)
+            {
+                EditorGUILayout.HelpBox("No input system is enabled, or you need to specify the type in the main configuration profile.", MessageType.Error);
+                return;
+            }
+
+            if (MixedRealityManager.Instance.ActiveProfile.InputActionsProfile == null)
```

Comment:
Why can't I use speech without input actions? The `SpeechEventData` provides the recognized text, so there are other ways to tell what was recognized.

Reply:
Because Input Actions are required for using the mrtk input system

Reply:
I think it's fair to say, some speech is pass through, words to be displayed.
Some are commands, which are actions to be performed. 

I think that is what you are getting at @keveleigh, correct?

If so, that works be a different pr, as this is just bug fixes and what you are suggesting would be a change.